### PR TITLE
Fix anime series not being matched to found results

### PR DESCRIPTION
**Pull request by medariox**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/generic_provider.py [hunk 1]
```diff
@@ -650,19 +660,17 @@ def _create_anime_search_string(self, show_scene_name, episode, search_string, a
         episode_string = show_scene_name + self.search_separator
 
         # If the show name is a season scene exception, we want to use the indexer episode number.
-        if (episode.scene_season > 1 and
+        if (episode.scene_season > 0 and
```
Comment (p0psicles) [inline medusa/providers/generic_provider.py]: I think my logic here was that there are only season scene exceptions from season 2?

Comment (medariox) [reply medusa/providers/generic_provider.py]: Yeah, from what I could see in the cache.db there are quite a few actually.

Comment (p0psicles) [reply medusa/providers/generic_provider.py]: Ow oops

File: medusa/scene_exceptions.py [hunk 2]
```diff
@@ -138,51 +139,25 @@ def get_all_scene_exceptions(series_obj):
 def get_scene_exceptions_by_name(show_name):
     """Look for a series_id, season and indexer for a given series scene exception."""
     # TODO: Rewrite to use exceptions_cache since there is no need to hit db.
-    # TODO: Make the query more linient. For example. `Jojo's Bizarre Adventure Stardust Crusaders` will not match
-    # while `Jojo's Bizarre Adventure - Stardust Crusaders` is available.
     if show_name is None:
         logger.debug('Scene exception lookup failed because no show name was provided')
         return [(None, None, None)]
 
+    normed_name = re.sub(r'\W+', '%', show_name, flags=re.UNICODE).strip('%')
```
Comment (p0psicles) [inline medusa/scene_exceptions.py]: Where not comparing using the sanitized name anymore?

**Pull request merged**